run.py

```diff
@@ -1,29 +1,21 @@
+#!/usr/bin/env python
+
 import click
 import ImmunoPDB
 import ab_haddock_format
 from pathlib import Path
 from pdbtools import pdb_tidy
 
 
-def tidy_up(file: Path, where: Path):
-    with file.open("r") as fh:
-        with where.open("a") as f:
-            new_pdb = pdb_tidy.run(fh, False)
-            n = 5000
-            try:
-                _buffer = []
-                _buffer_size = n
-                for lineno, line in enumerate(new_pdb):
-                    if not (lineno % _buffer_size):
-                        f.write(''.join(_buffer))
-                        _buffer = []
-                    _buffer.append(line)
-                f.write(''.join(_buffer))
-            except IOError:
-                pass
+def tidy_up(file: Path, where: Path) -> Path:
+    with where.open("w") as f:
+        new_pdb = pdb_tidy.run(file.open("r"), False)
```

Comment:
I suggest here to use `pdb-tidy --strict` to avoid introducing `TER` statements within one chain in case of missing loops for example.

Reply:
@amjjbonvin thank you for the suggestion, I switched to the strict mode.